examples/yoti_example_flask/app.py

```diff
@@ -49,4 +46,4 @@ def auth():
 
 
 if __name__ == '__main__':
-    app.run(host="0.0.0.0", debug=True)
+    app.run(host="0.0.0.0")
```

Comment:
debug doesn't work when using Python 2.7. This is just the debug command sent when using the `werkzeug` package, which provides more output from this package, we can still debug fine with Flask. I attempted to switch to a different command here, but this then breaks the Docker process